ns/rdf-xsd.ttl

```diff
@@ -0,0 +1,283 @@
+
+xsd:gYear a rdfs:Datatype ;
+  rdfs:isDefinedBy <http://www.w3.org/2001/XMLSchema#> ;
+  rdfs:label "gYear" ;
+  rdfs:comment "Gregorian year. Lexical form: CCYY (optionally with a leading '-' for years BCE and optional timezone)." .
+
+xsd:gMonth a rdfs:Datatype ;
+  rdfs:isDefinedBy <http://www.w3.org/2001/XMLSchema#> ;
+  rdfs:label "gMonth" ;
+  rdfs:comment "Gregorian month recurring each year. Lexical form: --MM with optional timezone." .
+
+xsd:gDay a rdfs:Datatype ;
+  rdfs:isDefinedBy <http://www.w3.org/2001/XMLSchema#> ;
+  rdfs:label "gDay" ;
+  rdfs:comment "Day in a month recurring each month. Lexical form: ---DD with optional timezone." .
+
+xsd:gYearMonth a rdfs:Datatype ;
+  rdfs:isDefinedBy <http://www.w3.org/2001/XMLSchema#> ;
+  rdfs:label "gYearMonth" ;
+  rdfs:comment "Gregorian year and month. Lexical form: CCYY-MM with optional timezone." .
+
+xsd:gMonthDay a rdfs:Datatype ;
+  rdfs:isDefinedBy <http://www.w3.org/2001/XMLSchema#> ;
+  rdfs:label "gMonthDay" ;
+  rdfs:comment "Month and day recurring each year. Lexical form: --MM-DD with optional timezone." .
+
+xsd:duration a rdfs:Datatype ;
+  rdfs:isDefinedBy <http://www.w3.org/2001/XMLSchema#> ;
+  rdfs:label "duration" ;
+  rdfs:comment "Time durations per ISO 8601 (PnYnMnDTnHnMnS), optionally with a leading '-'. Value space pairs a months component and a seconds component; only a partial order is defined." .
+
+xsd:yearMonthDuration a rdfs:Datatype ;
+  rdfs:subClassOf xsd:duration ;
+  rdfs:isDefinedBy <http://www.w3.org/2001/XMLSchema#> ;
+  rdfs:label "yearMonthDuration" ;
+  rdfs:comment "Duration restricted to year and month fields (seconds component is 0). Totally ordered; lexical form uses PnYnM with optional leading '-'." .
+
+xsd:dayTimeDuration a rdfs:Datatype ;
+  rdfs:subClassOf xsd:duration ;
+  rdfs:isDefinedBy <http://www.w3.org/2001/XMLSchema#> ;
+  rdfs:label "dayTimeDuration" ;
+  rdfs:comment "Duration restricted to day, hour, minute, second fields (months component is 0). Totally ordered; lexical form PnDTnHnMnS with optional leading '-'." .
+
+xsd:byte a rdfs:Datatype ;
+  rdfs:subClassOf xsd:short ;
+  rdfs:isDefinedBy <http://www.w3.org/2001/XMLSchema#> ;
+  rdfs:label "byte" ;
+  rdfs:comment "8-bit signed integer. Value space: [-128 .. 127]." .
+
+xsd:short a rdfs:Datatype ;
+  rdfs:subClassOf xsd:int ;
+  rdfs:isDefinedBy <http://www.w3.org/2001/XMLSchema#> ;
+  rdfs:label "short" ;
+  rdfs:comment "16-bit signed integer. Value space: [-32768 .. 32767]." .
+
+xsd:int a rdfs:Datatype ;
+  rdfs:subClassOf xsd:long ;
+  rdfs:isDefinedBy <http://www.w3.org/2001/XMLSchema#> ;
+  rdfs:label "int" ;
+  rdfs:comment "32-bit signed integer. Value space: [-2147483648 .. 2147483647]." .
+
+xsd:long a rdfs:Datatype ;
+  rdfs:subClassOf xsd:integer ;
+  rdfs:isDefinedBy <http://www.w3.org/2001/XMLSchema#> ;
+  rdfs:label "long" ;
+  rdfs:comment "64-bit signed integer. Value space: [-9223372036854775808 .. 9223372036854775807]." .
```

Comment:
Why do several descriptions have value space and no lexical space, and others have lexical space and no value space? I'm thinking that all should have both, even if some "values" of either might be "undefined" or similar handwave.